base/Dockerfile

```diff
@@ -10,10 +10,8 @@ ENV PATH /opt/conda/bin:$PATH
 ENV PYTHON_VERSION=${python}
 ENV DASK_VERSION=${release}
 
-RUN conda install --yes nomkl cytoolz cmake \
-    && conda install --yes mamba \
+RUN conda install --yes -c conda-forge nomkl cytoolz cmake mamba python=${PYTHON_VERSION} \
```

Comment:
I moved the python install up, and added `conda-forge` to this line to make sure we don't have different channels.

Reply:
hmm that doesn't seem to have quite done it.